docs/docs/concepts/redirects.mdx

```diff
@@ -1,9 +1,152 @@
 ---
 id: redirects
-title: Redirects and Flow Completion
+title: Browser Redirects and Flow Completion
 ---
 
-While we are working on improving the Ory Cloud documentation, please head over
-to our open source documentation for a deep-dive on this topic:
+Ory Cloud offers a few methods to integrate your own user interface (UI) such as
+utilising the SDK in Single Page Applications (ReactJs, Angular, VueJs etc.) or
+using redirects in Server Side Applications (NodeJs, PHP, Golang etc.).
+
+On this page of the documentation we will be covering Browser Redirects and how
+to configure it on the Ory Cloud console. If you are interested in utilising the
+SDK for Single Page Applications, check out our [SDK](./services-api.mdx)
+documentation.
+
+For a deeper dive into the background of this setup, head over to our open
+source documentation:
 
 - https://www.ory.sh/kratos/docs/concepts/browser-redirect-flow-completion
+
+## Allow list
+
+The Ory Cloud allows dynamic redirects to be set using the `?return_to=` query
+parameter on [Self-Service Flows](./self-service.mdx). As an example a user
+opens a sharable link to go to https://myapp.com/posts, however, this url
+requires the user to be signed in and thus redirects the user back to the login
+page. To return the user back to the original url we can append
+`?return_to=https://myapp.com/posts` to the specified Self-Service initialise
+flow url.
+
+The `Allow list` is used in this scenario to prevent Open Redirect Attacks by
+only allowing certain domains, or paths on a domain. As shown in this example
+below the https://sub.domain.myapp.com/only/path needs to match perfectly for
+this sub-domain and path for it to be allowed. Other redirects using myapp.com
+will fail, however, for https://anotherapp.com will succeed on any path.
+
+![Browser Redirects Allow List](../images/browser-redirects/browser-redirects-allow-list.png)
+
+## Redirect Flows
+
+The Ory Cloud has a total of 6 flows, Login, Registration, Verification,
```

Comment:
If configuration options are 1:1 to kratos then we could link hereto OS Kratos [docs](https://www.ory.sh/kratos/docs/concepts/browser-redirect-flow-completion/#redirection)  for more details

Reply:
It's not 1:1 since Kratos uses a base url which is the same as our Application URL.
e.g. https://some-slug.projects....
so this will be the base url for relative paths. So I updated it and made the Global URL the base url. So it won't be a 1:1 with open source.